Gems/GradientSignal/Code/Source/Components/LevelsGradientComponent.cpp

```diff
@@ -187,6 +185,21 @@ namespace GradientSignal
         return output;
     }
 
+    void LevelsGradientComponent::GetValues(AZStd::span<AZ::Vector3> positions, AZStd::span<float> outValues) const
+    {
+        if (positions.size() != outValues.size())
+        {
+            AZ_Assert(false, "input and output lists are different sizes (%zu vs %zu).", positions.size(), outValues.size());
+            return;
+        }
+
+        m_configuration.m_gradientSampler.GetValues(positions, outValues);
+
+        GetLevels(outValues, 
+                m_configuration.m_inputMid, m_configuration.m_inputMin, m_configuration.m_inputMax,
+                m_configuration.m_outputMin, m_configuration.m_outputMax);
```

Comment:
Unfortunately not, because this is also called from GradientSampler. 

At some point I'd like to circle back to both this and GradientSampler and change these runtime configuration fixups to only happen at init and value changes, not on every single call. (Because the GetValues() version only does it once for the entire call, it's a lower priority for me, but still bugs me.)

I also need to see what happens if I change GradientSampler to call the GetLevels(span) version, but it's got some tradeoffs to it (potentially 3x the loops), so I'm not doing that quite yet either.